### DISCUSSION TITLE
Does it support removing strikethrough characters from a cell?

### DISCUSSION THREAD

**Discussion by Rajesh-NLP** [mhammond/pywin32#2537] [General]
I want to remove all strikethrough characters from my excel file. VBA already resolves this request, but I cannot find any solution based on python (pywin32). The complete description of the very similar problem is described in the linked link. Kindly help me to find a solution to [this problem](https://stackoverflow.com/questions/67878012/possible-to-remove-all-strikethrough-content-from-within-each-cell-of-an-excel-w).

**Comment by mhammond** [comment]
Please see https://github.com/mhammond/pywin32#support for support requests.
